### PR TITLE
Only allow admins to search for untrusted users

### PR DESCRIPTION
**Pull request by mwiencek**
This flag is only supposed to be set by and visible to account admins, so this can be considered a privacy breach. cc @Freso who alerted this to me.

### CODE REVIEW

**Comment by gentlecat** [comment]
LGTM

**Comment by uklauer** [comment]
It is supposed? Why?

File: lib/MusicBrainz/Server/EditSearch/Predicate/EditorFlag.pm [hunk 1]
```diff
@@ -1,19 +1,44 @@
...
+    isa => 'Editor',
+    is => 'ro',
+    required => 1,
+);
+
+sub valid_flags {
+    my ($self) = @_;
+
+    my @flags = @{ $self->sql_arguments->[0] };
+    unless ($self->user->is_admin) {
```
Comment (uklauer) [inline lib/MusicBrainz/Server/EditSearch/Predicate/EditorFlag.pm]: If it is to be restricted, I don’t think it makes sense to show it to wiki transcluders, relationship editors etc. This should be `is_account_admin` instead (everywhere else in this PR, too).

(And unrelatedly, `is_admin` should probably be extended to test for account admins, too.)

Comment (uklauer) [inline lib/MusicBrainz/Server/EditSearch/Predicate/EditorFlag.pm]: If it is to be restricted, I don’t think it makes sense to show it to wiki transcluders, relationship editors etc. This should be `is_account_admin` instead (everywhere else in this PR, too).

(And unrelatedly, `is_admin` should probably be extended to test for account admins, too.)

**Comment by Freso** [comment]
We don't show the "Untrusted" flag on users profiles, so we shouldn't allow non-priv'd users to gain information about who's marked "Untrusted" by way of simply searching for edits by users with this flag.

**Pull request merged**